package.json

```diff
@@ -55,6 +55,7 @@
     "graphql-extensions": "file:packages/graphql-extensions"
   },
   "devDependencies": {
+    "@apollographql/graphql-upload-8-fork": "^8.1.3",
```

Comment:
I did this to make it visible to a build-time-only `import type` in apollo-server-fastify (I think that's the right way to do it?).